docs/installing.md

```diff
@@ -0,0 +1,18 @@
+# Installing Rancher Desktop
+
+Rancher Desktop is delivered as a desktop application. You can download it from
+the [releases page on GitHub](https://github.com/rancher-sandbox/rd/releases).
+
+When run for the first time or when changing versions, Kubernetes container
+images are downloaded. It may take a little time to load on first run for a new
+Kubernetes version.
+
+Note, the Windows installer is not yet signed. This will be coming soon. In the
+meantime, when presenting with a dialog box like the following click on "More
+info".
+
+![](images/windows-notice-1.png)
+
+This will present you with an option to run the application like the following.
```

Comment:
"The following dialog will then give you the option to run the application:" ["prefer fewer shorter words, someone must have said at some point"]

Reply:
Fixed